tests/acceptance/ENFormCreateCept.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+$I = new AcceptanceTester( $scenario );
+
+$I->wantTo( 'Check ENForm creation works' );
+
+$I->amOnPage( '/wp-admin/post-new.php?post_type=p4en_form' );
+
+$I->fillField('log', 'admin');
```

Comment:
@angtheod you can use `loginAsAdmin` provided by wp-browser for logging in
Event better we have a custom function `loginAsAdminCached`